### PR TITLE
Update some platform lists and add new labels

### PR DESCRIPTION
**Pull request by Crinfarr**
I noticed while reading that some of these were out of date, so I fixed them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,12 +19,12 @@ A curated list of awesome packages and frameworks for implementing javascript ap
 
 ### Web
 
-* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (Windows, macOS, Linux)
-* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (macOS only)
+* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (cross-platform)
+* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (cross-platform)
```
Comment (styfle) [inline README.md:23]: Does this support Linux?

File: README.md [hunk 2]
```diff
@@ -19,12 +19,12 @@ A curated list of awesome packages and frameworks for implementing javascript ap
 
 ### Web
 
-* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (Windows, macOS, Linux)
-* [Electrino](https://github.com/pojala/electrino) - a runtime to build desktop apps using the system's own web browser (macOS only)
+* [Neutralino](https://github.com/neutralinojs/neutralinojs) - a runtime to build desktop apps using the system's own web browser (cross-platform)
```
Comment (styfle) [inline README.md:22]: What's the difference between "cross-platform" and "Windows, macOS, Linux"?

Comment (Crinfarr) [reply README.md:22]: more people search "cross platform desktop framework" than "windows mac and linux desktop framework"

Comment (styfle) [reply README.md:22]: I don't think so. Let's keep it as is. Imagine I want to search "macOS".

File: README.md (hunk 1 above)
Comment (Crinfarr) [reply README.md:23]: The documentation is nonexistent, i read through it a few times but i might be wrong

File: README.md (hunk 2 above)
Comment (Crinfarr) [reply README.md:22]: I'll edit this a bit and make a new PR then?

File: README.md (hunk 1 above)
Comment (styfle) [reply README.md:23]: So its not cross platform 😉

File: README.md (hunk 2 above)
Comment (styfle) [reply README.md:22]: Sure 👍

**Pull request closed without merging**